packages/@aws-cdk/aws-pipes-targets-alpha/README.md

```diff
@@ -122,3 +123,51 @@ const pipe = new pipes.Pipe(this, 'Pipe', {
     target: pipeTarget
 });
 ```
+
+### AWS Lambda Function
+
+A Lambda Function can be used as a target for a pipe. The Lambda Function will be invoked with the (enriched/filtered) source payload.
+
+```ts
+declare const sourceQueue: sqs.Queue;
+declare const targetFunction: lambda.IFunction;
+
+const pipeTarget = new targets.LambdaFunction(targetFunction,{});
+
+const pipe = new pipes.Pipe(this, 'Pipe', {
+    source: new SomeSource(sourceQueue),
+    target: pipeTarget
+});
+```
+
+Specifying the Invocation Type when the target Lambda Function is invoked:
```

Comment:
```suggestion
The target Lambda Function is invoked synchronously by default. You can also choose to  invoke the Lambda Function asynchronously by setting `invocationType` property to `FIRE_AND_FORGET`.
```

Reply:
adjusted that, thanks for the suggestion